### PR TITLE
NAS-110692 / 12.0 / Replication Task Info Collapses unexpectedly/automatically

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

**Comment by undsoft** [comment]
It's better when items are open and you're just looking at them. It's a bit awkward when you are filtering, especially if you start pressing left-right arrows.
I'm okay with all rows getting collapsed when user starts filtering, if that helps.

https://user-images.githubusercontent.com/1481270/119095173-9f550f80-ba1a-11eb-866a-5783e3005a46.mov

**Comment by RehanY147** [comment]
The expanded row doesn't close automatically anymore

File: src/app/pages/common/entity/entity-table/entity-table.component.ts [hunk 1]
```diff
@@ -327,12 +328,11 @@ export class EntityTableComponent implements OnInit, AfterViewInit, OnDestroy {
     this.erdService.attachResizeEventToElement('entity-table-component');
   }
 
-  getFilteredRows() {
+  setFilteredRows(isTrigeredByFilter = false) {
```
Comment (RehanY147) [inline src/app/pages/common/entity/entity-table/entity-table.component.ts:331]: Should have a return type

File: src/app/pages/common/entity/entity-table/entity-table.component.ts [hunk 2]
```diff
@@ -180,6 +180,7 @@ export class EntityTableComponent implements OnInit, AfterViewInit, OnDestroy {
 
   protected toDeleteRow: any;
   private routeSub: any;
+  private expandedRowIds: any[] = [];
```
Comment (RehanY147) [inline src/app/pages/common/entity/entity-table/entity-table.component.ts:183]: Could we be more specific for the type? `number` or `string` or either/or

**Comment by AlexKarpov98** [comment]
@undsoft , @RehanY147 Fixed above issues.

**Review by undsoft — APPROVED** [review]
Works well now.

**Pull request merged**